.gitignore

```diff
@@ -33,3 +33,5 @@ coverage
 
 # talisman
 .talismanrc
+
+.DS_Store
```

Comment:
It's helpful have `.DS_Store` in a global `.gitignore` file: https://stackoverflow.com/a/7335487